Content.Client/IconSmoothing/ClientRandomIconSmoothSystem.cs

```diff
@@ -0,0 +1,23 @@
+using Content.Shared.IconSmoothing;
+
+namespace Content.Client.IconSmoothing;
+
+public abstract class ClientRandomIconSmoothSystem : SharedRandomIconSmoothSystem
```

Comment:
```suggestion
public sealed class ClientRandomIconSmoothSystem : SharedRandomIconSmoothSystem
```
This took entirely too long for me to notice.

Reply:
Oh shit, i do it again